airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/poetry/publish/commands.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+"""
+Module exposing the format commands.
+"""
+from __future__ import annotations
+
+from typing import Optional
+
+import asyncclick as click
+from pipelines.cli.dagger_pipeline_command import DaggerPipelineCommand
+from pipelines.models.contexts.click_pipeline_context import ClickPipelineContext, pass_pipeline_context
+
+from .context import PyPIPublishContext
+from .pipeline import PublishToPyPI
+
+CONNECTOR_PATH_PREFIX = "airbyte-integrations/connectors"
+
+
+@click.command(cls=DaggerPipelineCommand, name="publish", help="Publish a Python package to PyPI.")
```

Comment:
```suggestion
@click.command(cls=DaggerPipelineCommand, name="publish", help="Publish a Python package to a registry.")
```

Even if your implementation is not supporting private registries  I believe we can consider it's agnostic of the use of PyPi as a registry. PyPi is just the default.